### PR TITLE
fix error & fatal in normal shutdown

### PR DESCRIPTION
**Pull request by WindzCUHK**
# Description

fix error & fatal in normal shutdown

1. remove
```
[ERR]:        (tenantd.go:149):        context canceled
[ERR]:        (tenantd.go:141):        context canceled
...
[FATAL]:        (/sd/workspace/src/github.com/AthenZ/athenz-client-sidecar/main.go:160):        [context canceled]
```
2. add
```
[INFO]:        Athenz client sidecar received signal: terminated
...
[INFO]:        Athenz client sidecar shutdown success
```

## Type of change

- [x] Bug fix
- [ ] New feature
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Non-code changes (update documentation, pipeline, etc.)

### Flags

- [ ] Breaks backward compatibility
- [ ] Requires a documentation update
- [ ] Has untestable code

---

## Checklist

- [x] Followed the guidelines in the CONTRIBUTING document
- [x] Added prefix `[skip ci]`/`[ci skip]`/`[no ci]`/`[skip actions]`/`[actions skip]` in the PR title if necessary
- [x] Tested and linted the code
- [x] Commented the code
- [x] Made corresponding changes to the documentation
- [x] Passed all pipeline checking

## Checklist for maintainer
- [x] Use `Squash and merge`
- [x] Double-confirm the merge message has prefix `[skip ci]`/`[ci skip]`/`[no ci]`/`[skip actions]`/`[actions skip]`
- [x] Delete the branch after merge

### CODE REVIEW

File: usecase/tenantd.go [hunk 1]
```diff
@@ -138,15 +138,23 @@ func (t *clientd) Start(ctx context.Context) chan []error {
 	if t.access != nil {
 		go func() {
 			for err := range t.access.StartAccessUpdater(ctx) {
-				glg.Error(err)
+				if err == ctx.Err() {
+					glg.Info("Stopped access token updater")
+				} else {
+					glg.Errorf("StartAccessUpdater error: %s", err.Error())
+				}
```
Comment (officialajkim) [inline usecase/tenantd.go:145]:
```suggestion
if err == ctx.Err() {
	glg.Info("Stopped access token updater")
	continue
}

glg.Errorf("StartAccessUpdater error: %s", err.Error())
```

まあ、Elseは避けてもいいかなと思いました。

File: usecase/tenantd.go [hunk 2]
```diff
@@ -138,15 +138,23 @@ func (t *clientd) Start(ctx context.Context) chan []error {
...
 
 	if t.role != nil {
 		go func() {
 			for err := range t.role.StartRoleUpdater(ctx) {
-				glg.Error(err)
+				if err == ctx.Err() {
+					glg.Info("Stopped role token updater")
+				} else {
+					glg.Errorf("StartRoleUpdater error: %s", err.Error())
+				}
```
Comment (officialajkim) [inline usecase/tenantd.go:157]: 上と同じく

**Review by officialajkim — COMMENTED** [review]
Quick return with less indent & same lines of code

File: usecase/tenantd.go (hunk 2 above)
Comment (WindzCUHK) [reply usecase/tenantd.go]: fixed in 5c50265

**Review by ssunorz — APPROVED** [review]

**Pull request merged**